packages/validation/package.json

```diff
@@ -19,7 +19,11 @@
     "testInvalid": "node dist/runners/RunCheckFolder.js ../../../lionweb-integration-testing/testset/invalid",
     "testInvalidWithLanguage": "node dist/runners/RunCheckFolderWithLanguage.js ../../../lionweb-integration-testing/testset/withLanguage/invalid/ ../../../lionweb-integration-testing/testset/withLanguage/myLang.language.json",
     "testValidWithLanguage": "node dist/runners/RunCheckFolderWithLanguage.js ../../../lionweb-integration-testing/testset/withLanguage/valid/ ../../../lionweb-integration-testing/testset/withLanguage/myLang.language.json",
-    "test": "mocha"
+    "test": "mocha",
+    "release": "npm publish",
+    "publish-local": "npm publish --registry http://localhost:4873",
```

Comment:
What are these “local” scripts for? If we expose it on `main` then we should at least document it.

Reply:
I use them to do a local `npm publish` with a local npm registry (in my case `verdaccio`). That way I can use an unpublished version in other projects, which is sometimes very handy. We should document this indeed.

Reply:
documented